google/utils.go

```diff
@@ -408,3 +365,252 @@ func extractFirstMapConfig(m []interface{}) map[string]interface{} {
 
 	return m[0].(map[string]interface{})
 }
+
+func expandAliasIpRanges(ranges []interface{}) []*computeBeta.AliasIpRange {
```

Comment:
Since these are only specific to instances/instance templates, I'd rather them be in resource_compute_instance.go (or a new file) instead of the provider-wide utils

Reply:
SGTM. I think I'd prefer a new file. `_helpers.go` and `_utils.go` patterns both exist, so maybe `compute_instance_helpers.go`.